pkg/Directory.Build.props

```diff
@@ -25,6 +25,8 @@
     <PackageProjectUrl>https://dot.net/ml</PackageProjectUrl>
     <PackageIconUrl>https://aka.ms/mlnetlogo</PackageIconUrl>
     <PackageReleaseNotes>https://aka.ms/mlnetreleasenotes</PackageReleaseNotes>
+    <!-- space separated -->
+    <PackageTags>ML.NET ML Machine Learning</PackageTags>
```

Comment:
[space separated](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs#L760-L765)
I figure "Machine" and "Learning" are probably still useful separately.

Reply:
What about "Microsoft" tag?

Reply:
It's in the "author" and "owner" - one would hope that is enough.
```xml
    <id>Microsoft.ML</id>
    <authors>Microsoft</authors>
    <owners>Microsoft</owners>
    <description>ML.NET is a cross-platform open-source machine learning framework which makes machine learning accessible to .NET developers.</description>
    <tags>ML.NET ML Machine Learning</tags>
```
Currently, we don't show up in the first page of 
https://www.nuget.org/packages?q=machine+learning
which may not be surprising at this point, but we aren't even here:
https://www.nuget.org/packages?q=microsoft+machine+learning
because apparently it's favoring "microsoft" over "machine learning" (or at least, not favoring "and")